fix_test_suite_to_work_with_python3.patch
